sdk/storage/Azure.Storage.DataMovement.Blobs/src/BlobContainerClientExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Storage.DataMovement;
+using Azure.Storage.DataMovement.Blobs;
+
+namespace Azure.Storage.Blobs
```

Comment:
Should this namespace be `Azure.Storage.Blobs.Specialized` or something?

Reply:
Do we have a reason to hide this in specialized even after someone has already included the datamovement package for their code that is dealing directly with blob clients?

Reply:
Oh you know what, I didn't realize this was in Azure.Storage.DataMovement.Blobs, my bad I only saw namespace Azure.Storage.Blobs